server/api/api/endpoints/nuclio.py

```diff
@@ -37,7 +39,7 @@ async def list_api_gateways(
     ).list_api_gateways(project)
```

Comment:
Another note on previously existing code - why is the router at `/projects/{project}/nuclio/api-gateways`? I feel the `nuclio` part is a bit redundant and you can work with `/projects/{project}/api-gateways`.

Reply:
@theSaarco I assume the initial intention behind the 'nuclio' prefix was to establish a sort of conceptual 'separation' to show that we should request nuclio api directly. 

cc: @liranbg